bee-message/tests/ed25519_address.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::prelude::*;
+use bee_packable::Packable;
+
+use core::{convert::TryInto, str::FromStr};
+
+const ED25519_ADDRESS: &str = "52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c649";
+const ED25519_ADDRESS_INVALID_HEX: &str = "52fdfc072182654f163f5f0f9a621d729566c74d10037c4d7bbb0407d1e2c64x";
```

Comment:
maybe use totally different hex strings instead of just changing the last few chars; easier to read